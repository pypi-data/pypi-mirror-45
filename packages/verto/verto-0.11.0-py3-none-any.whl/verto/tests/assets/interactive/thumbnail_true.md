{interactive slug="binary-cards" type="whole-page" thumbnail="interactives/binary-cards/img/binarycards.png"}
