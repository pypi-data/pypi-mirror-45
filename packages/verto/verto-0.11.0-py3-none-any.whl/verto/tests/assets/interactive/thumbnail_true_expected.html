<a class="btn btn-interactive-link" href="{% url 'interactive' slug='binary-cards' %}">
<img class="btn-interactive-link-image" src="{% static 'interactives/binary-cards/img/binarycards.png' %}" />
<div class="btn-interactive-whole-page-text">
Click to load binary-cards
</div>
</a>
